### DISCUSSION TITLE
Error: Connection closed. on some route

### DISCUSSION THREAD

**Discussion by autsada** [vercel/next.js#56505] [Help]
### Summary

I deployed my app on Vercel and I got an error on one specific route, other routes work fine. I tested on several browsers and all of them are having the same error, this is the error that appears in the console `Error: Connection closed.
    at c` 
I tested locally (`npm run build` and `npm start`) and the app works fine for all routes. I tried to upgrade next to the latest version (13.5.4) and also downgrade to several versions, but the error still exists. Has anyone faced this issue before?

<img width="1308" alt="vercel" src="https://github.com/vercel/next.js/assets/34464087/34cc94f1-0ce0-43b3-b109-24229a93c78c">

<img width="1308" alt="localhost" src="https://github.com/vercel/next.js/assets/34464087/b041ea34-b68d-4b73-b71f-851b075d978e">

### Additional information

```js
Error: Connection closed.
    at c
```

**Comment by razzeee** [comment]
Were you able to figure this out?

**Comment by autsada** [comment]
Yes, but I can't remember how.
